doc/advanced_topics.rst

```diff
@@ -0,0 +1,126 @@
+***************
+Advanced Topics
+***************
+
+This page of documentation is reserved for advanced topics in SmartRedis
+data structures that may not be needed for all users.
+
+.. _advanced_topics_dataset_aggregation:
+
+Dataset Aggregation
+===================
+
+In addition to the ability to work with individual datasets, SmartRedis offers
```

Comment:
This sentence seems like it got messed up a bit in the revision.  Maybe should be "In addition to the ability to work with individual datasets, SmartRedis enables managing lists of datasets and working with them collectively."

Reply:
Another good catch, thanks